config/routes.rb

```diff
@@ -1,10 +1,32 @@
 Rails.application.routes.draw do
   resources :resorts do
+     resources :questions, only: [:create, :show]
 
     get 'favorites', to: 'favorites#create', as: 'create_favorite'
     get 'favorites/delete', to: 'favorites#destroy', as: 'delete_favorite'
 
   end
+
+  
+  resources :questions, only: [] do
+    resources :answers, only: [:create]
+  end
+  # resources :questions do
```

Comment:
All experimented routes commented out..please keep for later reference. 

I will remember to delete it when we are near to finish the project.